gradient_utils/metrics/tensorboard/patch.py

```diff
@@ -93,5 +100,116 @@ def __init__(self, *args, **kwargs):
     logger.debug("patching %s.%s", module, "EventFileWriter")
 
 
-def on_new_logdir(logdir):
-    logger.debug(logdir)
+def on_new_logdir(logdir, settings):
+    logger.debug("watching %s", logdir)
+    watcher = LogdirWatcher(logdir, settings)
+    settings.tensorboard_watchers.append(watcher)
+    watcher.start()
+
+
+class LogdirWatcher(object):
```

Comment:
`object` is redundant in Python3 (apparently)

Reply:
have we officially dropped support for python2 in this repo?

Reply:
Yea, I did when I revamped the build system bc Python2 is EOL